apps/launchpad/app/routes/_app+/rewards.tsx

```diff
@@ -1,20 +1,121 @@
 import { Avatar, Button } from '@0xintuition/1ui'
+import {
+  fetcher,
+  GetFeeTransfersDocument,
+  GetFeeTransfersQuery,
+  GetFeeTransfersQueryVariables,
+  useGetFeeTransfersQuery,
+} from '@0xintuition/graphql'
 
 import { AggregateIQ } from '@components/aggregate-iq'
 import { ErrorPage } from '@components/error-page'
 import { SkillRadarChart } from '@components/skill-radar-chart'
+import { ZERO_ADDRESS } from '@consts/general'
+import { fetchPoints, fetchRelicPoints } from '@lib/services/points'
+import { LoaderFunctionArgs } from '@remix-run/node'
+import { useLoaderData } from '@remix-run/react'
+import { getUser } from '@server/auth'
+import { dehydrate, QueryClient, useQuery } from '@tanstack/react-query'
 import { skills } from 'app/data/mock-rewards'
 import { motion } from 'framer-motion'
 import { ChevronRight } from 'lucide-react'
+import { formatUnits } from 'viem'
+
+export async function loader({ request }: LoaderFunctionArgs) {
+  const user = await getUser(request)
+  const address = user?.wallet?.address
+
+  console.log('Debug - User:', user)
+  console.log('Debug - Address:', address)
+
+  const url = new URL(request.url)
+  const activityLimit = parseInt(url.searchParams.get('activityLimit') || '10')
+  const activityOffset = parseInt(url.searchParams.get('activityOffset') || '0')
+
+  const queryClient = new QueryClient()
+
+  if (address) {
+    await Promise.all([
+      queryClient.prefetchQuery({
+        queryKey: ['get-relic-points', { address }],
+        queryFn: async () => {
+          const response = await fetchRelicPoints(address.toLowerCase())
+          return response
+        },
+      }),
+      queryClient.prefetchQuery({
+        queryKey: ['get-points', { address }],
+        queryFn: async () => {
+          const response = await fetchPoints(address.toLowerCase())
+          return response
+        },
+      }),
+      await queryClient.prefetchQuery({
+        queryKey: ['get-protocol-fees', { address }],
+        queryFn: () =>
+          fetcher<GetFeeTransfersQuery, GetFeeTransfersQueryVariables>(
+            GetFeeTransfersDocument,
+            {
+              address,
+              cutoff_timestamp: 1733356800,
+            },
+          ),
+      }),
+    ])
+  }
+
+  return {
+    address,
+    dehydratedState: dehydrate(queryClient),
+    initialParams: {
```

Comment:
Did you have any trouble from omitting address from the initialParams? I know we have anyway from the loader but did this cause cache issues with the key?